Simulation.cpp

```diff
@@ -358,7 +358,7 @@ void Simulation::UpdateRoutesAndLocations()
 //    }
 
 #pragma omp parallel for shared(pedsToRemove, allRooms)
-     for (long unsigned int p = 0; p < allPeds.size(); ++p) {
+     for (size_t p = 0; p < allPeds.size(); ++p) {
```

Comment:
there is a little conflict here with @xuqiancheng. 

Apparently, in windows, this change does not work. See also [here](https://github.com/JuPedSim/jpscore/pull/442/commits/cd0571e5b586ef22e5abfe39abdbaf8d03e33dd3#diff-441e5250dd8ecdf163d639bb5ac99da4L453)

Reply:
So should we stay with int? Or should we wait till #440 is merged and see if this solves this?

Reply:
@chraibi The only reason i can think of why this does not work is if the library type `size_t` is not "visible" i.e. the STD headers on linux / windows differ a bit in what they expose coincidentally. Looking at `Simulation.h` and 'Simulation.cpp` I see that there is no include `std::map` or `std::shared_ptr` so even those types are only coincidentally visible. It is considered essential practice in C++ to include all types a cpp/h file needs explicitly and not rely on other headers to transitively supply the type.

To fix the size_t issue on windows (and in general)  do in simulation.cpp:
```cpp
#include <cstddef>
```